.travis.yml

```diff
@@ -1,7 +1,22 @@
-language: go
-go:
-- 1.12.x
-env:
-- GO111MODULE=on # needed even for Go 1.12
-script:
-- go test -v ./...
+dist: bionic
+addons:
+  apt:
+    packages:
+      - systemd-container
+      - qemu-utils
+      - yum
+      - gnupg2
+matrix:
+  include:
+    - language: go
+      go: 1.12.x
+      env: GO111MODULE=on # needed even for Go 1.12
+      script: go test -v ./...
+
+    - language: python
+      python: 3.7
+      env: PYTHONUNBUFFERED=1
+      script:
+        # ubuntu's rpm package sets dbpath to ~/.rpmdb, which makes rpm fail...
+        - sudo sh -c 'mkdir /etc/rpm; echo "%_dbpath /var/lib/rpm" > /etc/rpm/macros'
+        - sudo env "PATH=$PATH" "OSBUILD_TEST_BUILD_PIPELINE=tools/test_image_info/base-from-yum.json" python3 -m unittest tools.test_image_info
```

Comment:
I think this should be build-from-yum.json. The point being that that is specific to the host, and might be swapped out on a different CI or locally, so any requirements of the build image for the real build should be in each of the pipelines we test.

Reply:
I had trouble understanding which is which. Thanks :)